integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -43,10 +55,463 @@ class AuthorizationServerIT extends ITSupport {
+
+    @Test
+    void updateAuthorizationServerPolicyTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        Policy policy = client.instantiate(Policy)
+            .setType(PolicyType.OAUTH_AUTHORIZATION_POLICY)
+            .setName("Test Policy")
+            .setDescription("Test Policy")
+            .setPriority(1)
+            .setConditions(client.instantiate(PolicyRuleConditions)
+                .setClients(client.instantiate(ClientPolicyCondition).setInclude(["ALL_CLIENTS"])))
+        registerForCleanup(policy)
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://default"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+
+        Policy createdPolicy = createdAuthorizationServer.createPolicy(policy)
+        registerForCleanup(createdPolicy)
+
+        assertThat(createdPolicy, notNullValue())
+
+        createdPolicy.setName("Test Policy Updated")
+        createdPolicy.setDescription("Test Policy Updated")
+
+        Policy updatedPolicy = createdAuthorizationServer.updatePolicy(createdPolicy.getId(), createdPolicy)
+        assertThat(updatedPolicy, notNullValue())
+        assertThat(updatedPolicy.getId(), equalTo(createdPolicy.getId()))
+        assertThat(updatedPolicy.getName(), equalTo(createdPolicy.getName()))
+        assertThat(updatedPolicy.getDescription(), equalTo(createdPolicy.getDescription()))
+    }
+
+    @Test
+    void deleteAuthorizationServerPolicyTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        Policy policy = client.instantiate(Policy)
+            .setType(PolicyType.OAUTH_AUTHORIZATION_POLICY)
+            .setName("Test Policy")
+            .setDescription("Test Policy")
+            .setPriority(1)
+            .setConditions(client.instantiate(PolicyRuleConditions)
+                .setClients(client.instantiate(ClientPolicyCondition).setInclude(["ALL_CLIENTS"])))
+        registerForCleanup(policy)
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://default"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+
+        Policy createdPolicy = createdAuthorizationServer.createPolicy(policy)
+        registerForCleanup(createdPolicy)
+
+        assertThat(createdPolicy, notNullValue())
+
+        createdAuthorizationServer.deletePolicy(createdPolicy.getId())
+
+        sleep(2000)
```

Comment:
nit: if you need to sleep, add a comment explaining why

Reply:
will do.